tooling/scripts/test-categories.yaml

```diff
@@ -126,6 +126,7 @@ messaging-networking1:
   - kafka
   - messaging
   - nats
+  - splunk
```

Comment:
Could we please all prefer adding new tests to the category that took shortest to run in some recent CI run? In that way we can keep the category execution times ballanced.  E.g. if you check here https://github.com/apache/camel-quarkus/actions/runs/663670257 , you see that messaging-networking1 took 1h3m which is not currently the fastest one. core-main-validation is at 44m.

Reply:
^ that's not a blocker for this PR. It is rather something for the future.